collectd/files/write_graphite.conf

```diff
@@ -10,16 +10,18 @@
 LoadPlugin write_graphite
 
 <Plugin write_graphite>
- <Carbon>
-   Host "{{ collectd_settings.plugins.write_graphite.host }}"
-   Port "{{ collectd_settings.plugins.write_graphite.port }}"
-   Prefix "{{ collectd_settings.plugins.write_graphite.prefix }}."
-   Postfix "{{ collectd_settings.plugins.write_graphite.postfix }}"
-   Protocol "{{ collectd_settings.plugins.write_graphite.protocol }}"
-   EscapeCharacter "{{ collectd_settings.plugins.write_graphite.escapecharacter }}"
-   LogSendErrors {{ collectd_settings.plugins.write_graphite.logsenderrors|lower }}
-   SeparateInstances {{ collectd_settings.plugins.write_graphite.separateinstances|lower }}
-   StoreRates {{ collectd_settings.plugins.write_graphite.storerates|lower }}
-   AlwaysAppendDS {{ collectd_settings.plugins.write_graphite.alwaysappendds|lower }}
- </Carbon>
+{%- for node_name, graphite_node in collectd_settings.plugins.write_graphite.nodes.iteritems() %}
+ <Node "{{ node_name }}">
```

Comment:
Thanks, was not aware of that. I suppose I shouldn't be surprised - Debian tends to keep very old/stable versions around. Will attempt to update later to include a version check. Do you know if the old <Carbon> format still accepts multiple nodes? Or will it die if it gets more than one?